junit-platform-console/src/main/java/org/junit/platform/console/tasks/DiscoveryRequestCreator.java

```diff
@@ -55,6 +56,12 @@ LauncherDiscoveryRequest toDiscoveryRequest(CommandLineOptions options) {
 				"Scanning the classpath and using explicit selectors at the same time is not supported");
 			return createClasspathRootSelectors(options);
 		}
+		if (options.isScanModulepath()) {
+			Preconditions.condition(!options.hasExplicitSelectors(),
+				"Scanning the module-path and using explicit selectors at the same time is not supported");
+			return Collections.singletonList(DiscoverySelectors.selectModule("ALL-MODULES"));
+
```

Comment:
Can we reference the constant here? There's an extra blank line after the return.

Reply:
The constant is declared "private" in `JigsawUtils (9)`. For it to be used is must use `public` and present in both `JigsawUtils` versions.

Reply:
FYI, the `jar --update ... --release X` checks for some (all?) binaries API differences of types that override type in base. It fails to update the artifact when there's an issue.